docs/en/operation/Troubleshooting.md

```diff
@@ -241,6 +241,14 @@ The parsable datetime formats include below:
 1. `--start-time <datetime>` specifies a datetime before with the log files can be ignored.
 A log file will be ignored if the last modified time is before this start time.
 
+
+## Alluxio limitation on file names
```

Comment:
Thinking that will it be better to use ## Alluxio limitations + ### URI path limitations so that all API docs can be pointed to the ## Alluxio limitations